cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -333,12 +333,13 @@ pub enum Inst {
         dst: Reg,
     },
 
-    /// A binary XMM instruction with an 8-bit immediate: cmp (ps pd) imm (reg addr) reg
+    /// A binary XMM instruction with an 8-bit immediate: e.g. cmp (ps pd) imm (reg addr) reg
     XmmRmRImm {
         op: SseOpcode,
         src: RegMem,
         dst: Writable<Reg>,
         imm: u8,
+        w: bool,
```

Comment:
Doc comment and/or better name for this field perhaps? All the others are self-explanatory but I'm not sure what just `w` means without context...

Reply:
Also, on preview below, the raw bool params to the encoding functions are somewhat bothersome; can we make this an enum instead for more self-documenting goodness?

Reply:
Also, the tradition has to name this `is64` to indicate a 64-bits operation or something along those lines.